web/static/js/components/lobby_stage.jsx

```diff
@@ -6,11 +6,10 @@ import StageProgressionButton from "./stage_progression_button"
 import ShareRetroLinkModal from "./share_retro_link_modal"
 import * as AppPropTypes from "../prop_types"
 import styles from "./css_modules/centered_text.css"
+import { findFacilitatorName } from "../reducers/presences"
 
 const LobbyStage = props => {
-  const { progressionConfig, currentUser, presences } = props
```

Comment:
❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 ❤️ 🔥 🎆 👍 💯 !!

Reply:
wait! looks like we aren't passing `facilitatorName` to the lobby stage!
![image](https://user-images.githubusercontent.com/4386645/36064513-07c84930-0e5a-11e8-8f06-464d20a4ccb7.png)